hybrid/flow.py

```diff
@@ -260,6 +367,22 @@ def halt(self):
 Race = RacingBranches
 
 
+class Dup(Runnable, traits.SIMO):
+    """Duplicates input :class:`~hybrid.core.State`, n times, into output
```

Comment:
Why not Copy or Duplicate?

Reply:
I like Dup for now. It's probably temporary anyway.